### PR TITLE
Sample python script logging

### PR DESCRIPTION
**Pull request by mariomiola**
sample python script logging

### CODE REVIEW

**Review by TejashwaKVerma — DISMISSED** [review]

File: scala/loadadw/src/main/scala/example/checkRemoteBucket.scala [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/**
```
Comment (TejashwaKVerma) [inline scala/loadadw/src/main/scala/example/checkRemoteBucket.scala:1]: Do we need separate PR for this?

Comment (mariomiola) [reply scala/loadadw/src/main/scala/example/checkRemoteBucket.scala:1]: It was part of the prior PR, related to RP

**Review by TejashwaKVerma — APPROVED** [review]
